app/routes.py

```diff
@@ -270,6 +271,8 @@ def config():
 
         session['config'] = config_data
         return redirect(config_data['url'])
+    else:
+        return json.dumps({})
```

Comment:
We probably want to just redirect back to home in this case, right? The default behavior of updating the config (in a non-disabled environment) is to redirect back to home, so I think it should be the same in this case.